### PR TITLE
Past project with coing partner for coding review

### PR DESCRIPTION
**Pull request by braincee**
I fixed some errors in the style linter.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          <div class="product" id="tenor">
+            <div class="level">Model X</div>
+            <h2>$1500</h2>
+            <ol>
+              <li>960-watt audio system</li>
+              <li>Active road noise reduction</li>
+              <li>Best listening experience</li>
+              <li>Wirelss Gaming</li>
+            </ol>
+            <button class="btn">Order Now</button>
```
Comment (braincee) [inline index.html:91]: Please you can add an attribute to the button type

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          <div class="product" id="bass">
+            <div class="level">Roadster</div>
+            <h2>$1750</h2>
+            <ol>
+              <li>Spacious Cabin</li>
+              <li>17" Cinimatic display</li>
+              <li>Multi-device Bluetooth</li>
+              <li>Fast Charge Electronics</li>
+            </ol>
+            <button class="btn">Order Now</button>
```
Comment (braincee) [inline index.html:102]: Please you can add an attribute to the button type

File: index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          <div class="product" id="valve">
+            <div class="level">Model 3</div>
+            <h2>$2200</h2>
+            <ol>
+              <li>Dual Motor All-Wheel Drive</li>
+              <li>Aerodynamic Elements</li>
+              <li>Quality Wheels & Tyres</li>
+              <li>Freedom to Travel</li>
+            </ol>
+            <button class="btn">Order Now</button>
```
Comment (braincee) [inline index.html:113]: Please you can add an attribute to the button type

File: style.css [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #A5AA52;
```
Comment (braincee) [inline style.css]: Please you can change the hex color from #A5AA52 to #a5aa52

File: style.css [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+  max-width: 150px;
+  width: 100%;
+  height: 30px;
+  margin: 15px 0;
+  border: 0;
+  background-color: #767522;
+}
+
+#hero input[type='submit']:hover {
+  background-color: #A5AA52;
```
Comment (braincee) [inline style.css]: Please you can change the hexadecimal color from #A5AA52 to #a5aa52

File: style.css [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+  margin-top: 30px;
+}
+
+#features .icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+  color: #CDC776;
```
Comment (braincee) [inline style.css]: Please you can use a hexadecimal color from #CDC776 to #cdc776

File: style.css [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+.product > .level {
+  background-color: #ddd;
+  color: black;
+  padding: 15px 0;
+  width: 100%;
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.product > h2 {
```
Comment (braincee) [inline style.css]: Please you can make the selector ".product > h2" to come before selector "#hero > h2"

File: style.css [hunk 8]
```diff
@@ -0,0 +1,317 @@
...
+  background-color: #94934c;
+  padding: 20px;
+}
+
+footer > ul {
+  display: flex;
+  justify-content: flex-end;
+}
+
+footer > ul > li {
```
Comment (braincee) [inline style.css]: Please you can make selector "footer > ul > li " to come before selector ".product > ol > li"

**Review by braincee — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make all required changes.

Submit for a re-review when you are done.

Thanks 😁

**Review by aimemalaika — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
 - Nice use of inline comment ✅ 

## Required Changes ♻️

- [ ] Kindly request to fix linter errors as the style linter is not passing.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved :heavy_check_mark:
### Hi @braincee,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## (Highlights) Good Points: :+1:
- You documented your work professionally. :heavy_check_mark:
- Github flow is used correctly. :heavy_check_mark:
- All linters checks are ok. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- [ ] Kindly you can explain why the student had to do your required changes like "It would be better semantically to do .....".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by braincee** [comment]
Thanks very much, @aimemalaika and @Omar-Muhamad 

thank-you

**Pull request merged**